spec/root_spec.cr

```diff
@@ -24,16 +24,33 @@ module PlaceOS::Api
 
       it "constructs service versions" do
```

Comment:
this test passes fine when I do focus: true but not when I run the full suite any obvious reason for this?

Reply:
What happens if you do a `WebMock.reset`?

Reply:
I've tried that already!